db/R/query.format.vars.R

```diff
@@ -16,7 +18,7 @@ query.format.vars <- function(input.id,con,format.id){
   site.lat <- NULL
   site.lon <- NULL
   
-  if (missing(format.id)) {
+  if (is.na(format.id)) {
```

Comment:
@bcow just to check the check for missing is on the format.id being NA and the ID, Lat/Long are set to NULL as placeholders until you fill in their values here correct?  Any cases where it wont be NA but you wont already have lat/long?